crates/webidl/src/lib.rs

```diff
@@ -70,6 +72,45 @@ pub fn compile(webidl_source: &str) -> Result<String> {
     Ok(compile_ast(&ast))
 }
 
+/// ChromeOnly is for things that are only exposed to privileged code in Firefox.
+pub fn is_chrome_only(ext_attrs: &Vec<Box<ExtendedAttribute>>) -> bool {
+    ext_attrs.iter().map(|external_attribute| {
+        let mut ret = false;
+        match external_attribute {
```

Comment:
I believe if you explicitly deref (or double deref) here, we can avoid using box patterns. I'm not aware of the current stabilization status of box patterns, so I would rather avoid them if possible.

Reply:
I had quite a few weird ownership issues with a double deref, not sure what I was doing wrong and ended up on the box syntax in the end. Maybe you could show me an example of how to get it working?